examples/tagging.ini

```diff
@@ -0,0 +1,129 @@
+; This is an example configuration for training machine translation.  It is an
```

Comment:
This should also have a nice header as the ` examples/factored.ini ` - it is not an example configuration for MT.